App/app/lib/main.dart

```diff
@@ -1,10 +1,16 @@
+import 'dart:io';
+
```

Comment:
Remove all the changes from main.dart.
As i have allready implemnetd a multiprovider in my local and we will be using that.

Thanks :D

Reply:
Won't do

Reply:
Do it